pkg/sql/information_schema.go

```diff
@@ -100,6 +103,17 @@ func dIntFnOrNull(fn func() (int32, bool)) tree.Datum {
 	return tree.DNull
 }
 
+func validateInformationSchemaTable(table *sqlbase.TableDescriptor) error {
+	// Make sure no tables have boolean columns.
+	for _, col := range table.Columns {
+		if col.Type.SemanticType == sqlbase.ColumnType_BOOL {
+			return errors.Errorf("information_schema tables should never use BOOL columns. "+
```

Comment:
We started prefixing our assertion errors with `programming error:`. I think it is applicable here.

Reply:
Done.